README.rst

```diff
@@ -42,4 +42,235 @@ Other code (in ``libcore``, ``lib``, etc.) should not be used by external users
 .. |Python Version| image:: https://img.shields.io/pypi/pyversions/libcoveocds.svg
    :target: https://pypi.org/project/libcoveocds/
 
+Output JSON format
+------------------
 
+The output is an object with the following properties:
+
+===================================== ===================== ==============
+Property (key) name		      Type                  Value
+===================================== ===================== ==============
+``file_type``                         string                The type of the file supplied, e.g. ``json``
+``version_used``                      string                The version of the OCDS schemas used, e.g. ``1.1``
+``schema_url``                        string                The url to the package schema used, e.g. ``https://standard.open-contracting.org/1.1/en/release-package-schema.json``
+``extensions``                        object                An extensions_ object
+``validation_errors``                 array[object]         An array of validation_errors_ objects
+``common_error_types``                array[]               Always an empty array.
```

Comment:
@Bjwebb

Reply:
Because it's only used by other coves.

Reply:
Out of curiosity, what are the semantics in the other CoVEs?

Reply:
It's a [list of types and formats](https://github.com/ThreeSixtyGiving/dataquality/blob/02c6a70c30413c76234773afc46fa1a095abde5c/lib360dataquality/cove/threesixtygiving.py#L355-L362) to create a "more info" link for, in the validation error section of the CoVE web interface. These then link to a common errors page https://dataquality.threesixtygiving.org/common_errors#date-time. It's necessary because not all types and formats are on that page, so we don't want to make useless links.

Reply:
Aha. Yes, in the OCDS DRT we would prefer to have such instructions on the results page rather than a separate page, so OK to leave this as just "Always an empty array"